tools/upload_to_cloud.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import posixpath
+import spear
+import subprocess
+import sys
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cloud_path_prefix", required=True)
+    parser.add_argument("--version_tag", required=True)
+    parser.add_argument("--paks", action="store_true")
```

Comment:
The way `--paks` is used here won't scale if there are ever more than two different artifacts that we want to upload (i.e., executables, paks, and some other stuff). And there will be soon, because we will want to upload the results of our export pipeline. I recommend the following because then new artifacts can be added more easily without needing to change any surrounding logic.

```
    parser.add_argument("--upload_executable", action="store_true")
    parser.add_argument("--upload_paks", action="store_true")

    # ...

    if args.upload_executable:
        # ...
    if args.upload_paks:
        #
```

Reply:
Aah, yes, this makes sense. I'll update this.